Update repositories.txt

E-Lagori opened the pull request:
https://github.com/E-Lagori/ELi_McM_4_00 library removed

per1234 commented:
Hello @E-Lagori. I am happy to remove the library for you, but I would like to get some clarification before starting on that process (which might take a couple of weeks to complete due to the need for a manual database edit by someone who is currently on vacation).

I notice you added it only a few hours ago (https://github.com/arduino/library-registry/pull/1717), and then attempted to submit it redundantly only minutes ago (https://github.com/arduino/library-registry/pull/1718). This makes me think that you may actually be hoping to accomplish something other than a removal in the end. If so, please tell me what you want to accomplish. It may be that I can help you to do that in a more efficient manner.

E-Lagori commented:
Hi,Thank you very much for your reply. I added a new library to arduino few hours back. Now I updated the readme file with better documentation. I changed the library.properties to 1.0.1. when I pushed it to my git repository. I started getting several errors from lint action. So I thought of removing the link from the Arduino and reinserting it after all my corrections are done. If there is any better way of updating the library I would gladly do it. I am new to git so any help is appreciated.Thank youAnish bekalcontact ---- On Wed, 17 Aug 2022 11:14:04 +0530  ***@***.***  wrote ----
Hello @E-Lagori. I am happy to remove the library for you, but I would like to get some clarification before starting on that process (which might take a couple of weeks to complete due to the need for a manual database edit by someone who is currently on vacation).
I notice you added it only a few hours ago (#1717), and then attempted to submit it redundantly only minutes ago (#1718). This makes me think that you may actually be hoping to accomplish something other than a removal in the end. If so, please tell me what you want to accomplish. It may be that I can help you to do that in a more efficient manner.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

per1234 commented:
OK, thanks for the clarification. I identified the problem and have submitted a pull request to fix it: https://github.com/E-Lagori/ELi_McM_4_00/pull/2

Since there is now no need to remove the library, I will go ahead and close this pull request. I'll continue monitoring the thread so please let me know if you have any questions or concerns.

By the way, it's very cool that you are using [the `arduino/arduino-lint-action` GitHub Actions action](https://github.com/arduino/arduino-lint-action) in your library's CI system!
